packages/trader/src/Modules/CFD/Containers/cfd-dashboard.tsx

```diff
@@ -205,6 +204,10 @@ const CFDDashboard = (props: TCFDDashboardProps) => {
             setIsRealEnabled(true);
             setIsDemoEnabled(true);
         }
+        if (window.location.hash === '#demo') {
+            setIsDemoEnabled(true);
+            setActiveIndex(1);
+        }
```

Comment:
@balakrishna-binary no, we don’t need it, on the 'real' tab issues don’t appear, I've added check with hash as an additional condition only for 'demo'. I have blocked the card because of the backend, after soon fixing the problems with real/demo accounts from the backend, I will rewrite this part logic a little bit. For now, I think the additional condition for a check is the best way.